### PR TITLE
Pairwise Confusion

### PR DESCRIPTION
**Pull request by t-detlefsen**
Implement Pairwise confusion based on [Pairwise Confusion for Fine-Grained Visual Classification](https://www.ecva.net/papers/eccv_2018/papers_ECCV/papers/Abhimanyu_Dubey_Improving_Fine-Grained_Visual_ECCV_2018_paper.pdf)

### CODE REVIEW

**Comment by t-detlefsen** [comment]
- Modified `config.yaml`
    - Option to use classification or PC loss
    - Included PC hyperparameters
- Moved `PairwiseConfusion` and `EntropicConfusion` to `utils.py`
- Moved `create_experiment_directory` to `utils.py`
- Merged `resnet_cnn.py` and `resnet_pc.py`
- Sorted classnames in `dataloader.py` for consistency across systems
- Updated `README.md`

**Review by robertkim1221 — COMMENTED** [review]
👍
